docs/src/references/cli/tedge-http.md

```diff
@@ -0,0 +1,224 @@
+                            Example: 8000
+         http.bind.address  The address of the File Transfer Service HTTP server binds to for internal use. 
+                            Examples: 127.0.0.1, 192.168.1.2, 0.0.0.0
+```
+
+```sh title="Enabling TLS aka HTTPS"
+            http.cert_path  The file that will be used as the server certificate for the File Transfer Service. 
+                            Example: /etc/tedge/device-certs/file_transfer_certificate.pem
+             http.key_path  The file that will be used as the server private key for the File Transfer Service. 
+                            Example: /etc/tedge/device-certs/file_transfer_key.pem
+```
+
+```sh title="Enforcing mTLS"
+              http.ca_path  Path to a directory containing the PEM encoded CA certificates that are trusted when checking incoming client certificates for the File Transfer Service. 
+                            Example: /etc/ssl/certs
+```
+
+### On the box running the cumulocity mapper
+
+The following `tedge config` settings control the access granted to child devices
+on the HTTP services provided by the Cumulocity mapper ([Cumulocity proxy](/references/cumulocity-proxy/)).
+This can be done along three security levels.
+
+```sh title="Listening HTTP requests"
+    c8y.proxy.bind.address  The IP address local Cumulocity HTTP proxy binds to. 
+                            Example: 127.0.0.1
+       c8y.proxy.bind.port  The port local Cumulocity HTTP proxy binds to. 
+                            Example: 8001
+```
+
+```sh title="Enabling TLS aka HTTPS"
+       c8y.proxy.cert_path  The file that will be used as the server certificate for the Cumulocity proxy. 
+                            Example: /etc/tedge/device-certs/c8y_proxy_certificate.pem
+        c8y.proxy.key_path  The file that will be used as the server private key for the Cumulocity proxy. 
+                            Example: /etc/tedge/device-certs/c8y_proxy_key.pem
+```
+
+```sh title="Enforcing mTLS"
+         c8y.proxy.ca_path  Path to a file containing the PEM encoded CA certificates that are trusted when checking incoming client certificates for the Cumulocity Proxy. 
+                            Example: /etc/ssl/certs
+```
+
+### One all boxes
+
+The following `tedge config` settings control how child devices access the local HTTP services.
+This has to be done in consistent way with the main agent and Cumulocity mapper settings.
+
+```sh title="Reaching local HTTP services"
+     c8y.proxy.client.host  The address of the host on which the local Cumulocity HTTP Proxy is running, used by the Cumulocity mapper. 
+                            Examples: 127.0.0.1, 192.168.1.2, tedge-hostname
+     c8y.proxy.client.port  The port number on the remote host on which the local Cumulocity HTTP Proxy is running, used by the Cumulocity mapper. 
+                            Example: 8001
```

Comment:
`http.client.host` and `http.client.port` are missing.

Reply:
added https://github.com/thin-edge/thin-edge.io/pull/3357/commits/f71705349e51b269ab232631e3b7417334a342d8